pkg/activator/handler/handler.go

```diff
@@ -76,14 +77,15 @@ func New(l *zap.SugaredLogger, r activator.StatsReporter, t *activator.Throttler
 		}
 	}
 	return &activationHandler{
-		logger:         l,
-		transport:      network.AutoTransport,
-		reporter:       r,
-		throttler:      t,
-		revisionLister: rl,
-		sksLister:      sksL,
-		serviceLister:  sl,
-		probeTimeout:   defaulTimeout,
+		logger:          l,
+		transport:       network.AutoTransport,
+		reporter:        r,
+		throttler:       t,
+		revisionLister:  rl,
+		sksLister:       sksL,
+		serviceLister:   sl,
+		probeTimeout:    defaulTimeout,
+		endpointTimeout: defaulTimeout,
```

Comment:
I am not sure this is right. I.e. user function may take infinitely long, in theory.

Reply:
Good point, we do need to think about this a bit, but even if a user function takes infinitely long this shouldn't be an issue if we can scale up more endpoints. Really its request timeout when waiting on scale up / resource freeing up.

Maybe this should be a configurable? I'd say per app but it also seems like operators would want to set this value to prevent DOS/blowing up activators when apps hit max scale.

Reply:
It can be configurable, but I am not sure if there can be one good value even within a single consumer project. 
So, I think for now we should just agree on a const value, which we consider reasonable and see if that causes problems in future.

Reply:
Agree - do you think 2min is a reasonable const? :)

Reply:
Sure :-)